### DISCUSSION TITLE
Postal Bounce Msg Back to sender

### DISCUSSION THREAD

**Discussion by Ranmoodk** [postalserver/postal#1656] [Help with using Postal]
Hi i am stuck in one issue postal is not sending bounce msg to sender. Can u Guys Please help me out to fix it out i am using multiple domaind

**Comment by willpower232** [comment]
Postal does not return the message to the original sender as the sender is likely to be a program rather than a person so if you want to monitor what happens to messages then you can make use of webhooks to see what happens to messages after your application sends them.
